frontend/src/components/login/signup.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { Link, Redirect } from "react-router-dom";
+import axios from 'axios';
+import { Card, Form, Input, Button, Error } from "./AuthForm";
+import { useAuth } from "../context/auth";
+
+function Signup({toggleModalBox, setDisplayModal}) {
+  const [isLoggedIn, setLoggedIn] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [username, setUserName] = useState("");
+  const [password1, setPassword1] = useState("");
+  const [password2, setPassword2] = useState("");
+  const email = ""
+  const { setAuthTokens } = useAuth();
+
+  function postSignup() {
+    const http = axios.create({
+      baseURL: process.env.REACT_APP_DJANGO_API,
+      responseType: 'json',
+    });
+    http.post("/rest-auth/registration/", {
```

Comment:
Here we use the API client, but it would be nice not to hardcode any API url in the front code, and organize it like we started in the API client file.